### ISSUE TITLE
Bug on getAllNewMessages:  Cannot read property '_models' of undefined.

### ISSUE THREAD

**Issue by sandro-salles** [open-wa/wa-automate-nodejs#560]
**Describe the bug**
getAllNewMessages throws a Cannot read property '_models' of undefined error.

**Steps to Reproduce**
1. Create client
2. Upon successful connection, call client.getAllNewMessages
3. Error is thrown like this:

> Error: Evaluation failed: TypeError: Cannot read property '_models' of undefined\n    at /Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/lib/wapi.js:300:113\n    at Array.map (<anonymous>)\n    at Object.window.WAPI.getAllNewMessages (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/lib/wapi.js:300:97)\n    at __puppeteer_evaluation_script__:1:28\n    at ExecutionContext._evaluateInternal (/Users/sandrosalles/workspace/zappo/client/node_modules/puppeteer/lib/ExecutionContext.js:102:19)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)\n    at async ExecutionContext.evaluate (/Users/sandrosalles/workspace/zappo/client/node_modules/puppeteer/lib/ExecutionContext.js:33:16)\n  -- ASYNC --\n    at ExecutionContext.<anonymous> (/Users/sandrosalles/workspace/zappo/client/node_modules/puppeteer/lib/helper.js:94:19)\n    at DOMWorld.evaluate (/Users/sandrosalles/workspace/zappo/client/node_modules/puppeteer/lib/DOMWorld.js:89:24)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)\n  -- ASYNC --\n    at Frame.<anonymous> (/Users/sandrosalles/workspace/zappo/client/node_modules/puppeteer/lib/helper.js:94:19)\n    at Page.evaluate (/Users/sandrosalles/workspace/zappo/client/node_modules/puppeteer/lib/Page.js:612:14)\n    at Page.<anonymous> (/Users/sandrosalles/workspace/zappo/client/node_modules/puppeteer/lib/helper.js:95:27)\n    at Client.<anonymous> (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:1219:46)\n    at step (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:44:23)\n    at Object.next (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:25:53)\n    at /Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:19:71\n    at new Promise (<anonymous>)\n    at __awaiter (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:15:12)\n    at Client.getAllNewMessages (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:1213:16)\n  -- ASYNC --\n    at Page.<anonymous> (/Users/sandrosalles/workspace/zappo/client/node_modules/puppeteer/lib/helper.js:94:19)\n    at Client.<anonymous> (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:1219:46)\n    at step (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:44:23)\n    at Object.next (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:25:53)\n    at /Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:19:71\n    at new Promise (<anonymous>)\n    at __awaiter (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:15:12)\n    at Client.getAllNewMessages (/Users/sandrosalles/workspace/zappo/client/node_modules/@open-wa/wa-automate/dist/api/Client.js:1213:16)\n    at on_initialize (file:///Users/sandrosalles/workspace/zappo/client/tasks.js:188:16)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)

**create() code**
This is the code you use to create the client. e.g
```javascript
create({
          sessionId: `mySessionId`,
          sessionDataPath: 'sessions',
          chromiumArgs,
          sessionData,
          headless: false,
          autoRefresh: true,
          blockCrashLogs: true,
          cacheEnabled: true,
          throwErrorOnTosBlock: true,
          logConsole: false,
          logConsoleErrors: true,
          skipBrokenMethodsCheck: true,
          useChrome: true,
          qrLogSkip: true,
          bypassCSP: true,
          disableSpins: true,
          useStealth: false,
          killProcessOnBrowserClose: false,
          qrTimeout: config.WA_QR_TIMEOUT,
          authTimeout: config.AUTH_TIMEOUT,
          proxyServerCredentials
        })
        .then((client) => {
          ...
        })
```

**Expected behavior**
getAllNewMessages to actually return the new messages (if any).

**DEBUG INFO**
This is the info printed to the console when you start your app. It should look like this
```
Debug Info {
  WA_VERSION: 2.2025.6,
  PAGE_UA: WhatsApp/2.2019.8 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36,
  WA_AUTOMATE_VERSION: 1.9.75,
  BROWSER_VERSION: Chrome/83.0.4103.106
}
```

**Host (please complete the following information):**
 - OS: Docker container (node:12.18.0-buster-slim) 

Regards

**Comment by smashah** [comment]
Thanks for reporting @sandro-salles will fix this ASAP

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.76](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.75...1.9.76) (2020-06-20)

- Release 1.9.76 [`fb16d29`](https://github.com/open-wa/wa-automate-nodejs/commit/fb16d29c81aff67479b08d0070a7356254572900)
- fixed `getAllNewMessages` #560 [`d290036`](https://github.com/open-wa/wa-automate-nodejs/commit/d290036d0f342d8e1992ba50854406d0667104dd)
- migrated license and broken method server #499 [`5858a97`](https://github.com/open-wa/wa-automate-nodejs/commit/5858a97ad8974160e18c6d70385e083ef148d639)

**Comment by smashah** [comment]
@sandro-salles all working now
